HACKING.rst

```diff
@@ -0,0 +1,111 @@
+=======
+HACKING
+=======
+
+Getting Started
+---------------
+
+Getting the code
+~~~~~~~~~~~~~~~~
+
+If you just want to look at the code, you can just clone the main repository:
+
+::
+    $ git clone git@github.com:juju/theblues.git
```

Comment:
Just wondering, is 'theblues' still a good project name now that it is being exposed to the world? In isolation it doesn't make a lot of sense.